mobile/examples/object_detection/ios/Podfile

```diff
@@ -1,11 +1,12 @@
-# Uncomment the next line to define a global platform for your project
-# platform :ios, '9.0'
-
-source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '11.0'
 
 target 'ORTObjectDetection' do
-  # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
+
   # Pods for ORTObjectDetection
-  pod 'onnxruntime-mobile-objc', '1.8.1'
+  pod 'onnxruntime-mobile-objc'
```

Comment:
not pinning to a specific version (and removed Podfile.lock) so we can always test the latest release